composer.json

```diff
@@ -17,26 +17,29 @@
     ],
     "require": {
         "php": "^7.1",
+        "alcaeus/mongo-php-adapter": "^1.1",
```

Comment:
This 100% didn't show up in the PR before I merged it. :see_no_evil:

Reply:
Maybe it got there while you were testing my proposed fix for phpunit v6.

Reply:
I've hot-fixed it on master.

Reply:
Don't you actually need that in order to use ODM now? We require PHP 7.1 which doesn't support the dated mongo extension anyway.

Reply:
Yes, but we're only suggesting ODM bundle, not requiring it. It's up to the end-user to install the ODM bundle and, by which, this adapter.

Reply:
Yea, you are right. Even if the end-user requires that workaround to use ODM, this is not the correct bundle to take care of the problem.

Reply:
If you're not using Mongo ODM, no reason to have this installed.

Reply:
yeah, this should be moved to the `require-dev` section